doc/releases/changelog-0.36.0.md

```diff
@@ -79,48 +90,88 @@
 
   @qml.qnode(dev)
   def circuit():
-
-      generator(wires=range(3))
-      qml.AmplitudeAmplification(U, O, iters=5, fixed_point=True, work_wire=3)
+      generator(wires=range(3)) # prepares |Psi> = U|0>
```

Comment:
Suggestion for line 88:
````md
  ```

  Here `U` is a quantum operation that is created by decorating a quantum function
  with `@qml.prod`. This could alternatively be done by creating a user-defined
  [custom operation](https://docs.pennylane.ai/en/stable/development/adding_operators.html)
  with a decomposition. Amplitude amplification can then be set up within a circuit:

  ```python
````